base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
It is very relevant since you do not use this function to convert an object `y` to `T` for passing to C by calling `unsafe_convert(T, y)`, which is what the current doc string `` Convert `x` to a value of type `T` `` suggests. You first do `x = cconvert(T, y)` and then call `unsafe_convert(T, x)`.

The type of `x` (and the value of `x`) is basically never what the user should worry/care about and it's one of the most important thing about `unsafe_convert` and `cconvert` that they are used this way (i.e. the input to `unsafe_convert` is the output of `cconvert`). It's not allowed to call `unsafe_convert` with `x` being anything that's not the return value of `cconvert`.